rebuild/src/public_key.h

```diff
@@ -0,0 +1,96 @@
+#ifndef BLST_TS_PUBLIC_KEY_H__
+#define BLST_TS_PUBLIC_KEY_H__
+
+#include <memory>
+
+#include "addon.h"
+#include "blst.hpp"
+#include "napi.h"
+
+#define BLST_TS_PUBLIC_KEY_LENGTH_COMPRESSED 48U
```

Comment:
Same comment as in the secret key header file. It might make sense to use static variables instead.